src/interfaces.d.ts

```diff
@@ -35,18 +44,18 @@ export interface HasSortEvent {
 }
 
 export interface HasItem {
-	item: ItemProperties<any>;
+	item: ItemProperties;
 }
 
 export interface HasItems {
-	items: ItemProperties<any>[];
+	items: ItemProperties[];
 }
 
 export interface HasValue {
 	value: string;
 }
 
-export interface ItemProperties<T> {
+export interface ItemProperties<T = any> {
```

Comment:
Right now, the object structure is simple, but `ItemProperties` adds much more moving forward.